tests/test_outputs/test_form_output_prep.py

```diff
@@ -0,0 +1,361 @@
+"""Tests for 'form_output_prep.py'."""
+# Standard Library Imports
+import pytest
+
+# Third Party Imports
+import pandas as pd
+import numpy as np
+
+# Local Imports
+from src.outputs.form_output_prep import form_output_prep
+
+class TestFormOutputPrep(object):
+    """Tests for form_output_prep."""
+
+    @pytest.fixture(scope="function")
+    def estimated_df(self) -> pd.DataFrame:
+        """'estimated_df' input for form_output_prep.
+        
+        NOTE: This is a subset of columns for testing purposes
+        """
+        columns = [
+            "reference", 
+            "instance", 
+            "period",
+            "201",
+            "202",
+            "211",
+            "602", 
+            "604", 
+            "a_weight", 
+            "formtype",
+            "pg_numeric"
+        ]
+        data = [
+            [39900000404, 0, 202212, 'AA', np.nan, np.nan, 100.0, 'Yes',
+                6.727272727272728, 6, 40],
+            [39900000404, 1, 202212, 'AA', 3146383.9595, 4628363.6364, 100.0,
```

Comment:
I'm wondering why you are choosing to use numbers like this 3146383.9595.... do you think the test would be easier to read and to follow what's happening if they were round numbers where you can do the sums in your head?

Reply:
This data is 'realistic', however I agree that using more round numbers would make the tests more readable for someone trying to follow them. Happy to change this if you agree.

Reply:
Updated these to round numbers.